hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/FreezeServiceImpl.java

```diff
@@ -48,13 +42,12 @@ private Schema networkAdminSchema() {
             @Override
             @SuppressWarnings("rawtypes")
             public Set<StateDefinition> statesToCreate() {
-                return Set.of(StateDefinition.singleton(UPGRADE_FILES_KEY, new MonoSpecialFilesAdapterCodec()));
+                return Set.of(StateDefinition.singleton(UPGRADE_FILE_HASH_KEY, new BytesCodec()));
             }
 
             @Override
-            public void migrate(@NonNull MigrationContext ctx) {
-                final var upgradeFilesState = ctx.newStates().getSingleton(UPGRADE_FILES_KEY);
-                upgradeFilesState.put(new MerkleSpecialFiles());
+            public void migrate(@NonNull final MigrationContext ctx) {
+                // nothing to do
             }
```

Comment:
I'm a little confused by this, but I may have missed some discussion

Reply:
The only datum that this service stores is the upgrade file hash. It doesn't make sense to migrate this datum, since it is discarded after each upgrade is launched.